### PR TITLE
refactor: update navigation paths for a more consistent experience [BD-38] [TNL-8429] [BB-4419]

### PR DESCRIPTION
**Pull request by xitij2000**
Navigation paths have now been structured such that you can move between tabs and topcis without losing the current page.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @xitij2000! I've created [OSPR-6023](https://openedx.atlassian.net/browse/OSPR-6023) to keep track of it in JIRA.

As a core committer in this repo, you can merge this once the pull request is approved per the [core committer reviewer requirements](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529675973/Open+edX+Core+Committers#Reviews-and-Ownership) and according to the agreement with your edX Champion.

**Review by tinumide — APPROVED** [review]
:+1:

- [x] I tested this PR and checked that navigating around works as expected, and when you move from topics to posts etc, the correct tab is highlighted.
- [x] I read through the code
- [x] I checked for accessibility issues
- [x] ~Includes documentation~ NA
- [x] ~I made sure any change in configuration variables is reflected in the corresponding client's `configuration-secure` repository.~ NA

File: src/discussions/topics/TopicsView.jsx
Comment (mehaknasir) [inline src/discussions/topics/TopicsView.jsx:14]: I think this is added by mistake

**Review by mehaknasir — APPROVED** [review]
looks good to me. added a minor comment to remove console statement

**Comment by edx-webhook** [comment]
@xitij2000 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/frontend-app-discussions&entry.867055334=https://github.com/edx/frontend-app-discussions/pull/14&entry.1484655318=https://github.com/xitij2000&entry.752974735=2021-09-03+14:31&entry.1917517419=2021-09-07+15:52&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**
